src/daft-functions/Cargo.toml

```diff
@@ -1,11 +1,29 @@
 [dependencies]
+arrow2 = {workspace = true, features = [
+  "chrono-tz",
+  "compute_take",
+  "compute_cast",
+  "compute_aggregate",
+  "compute_if_then_else",
+  "compute_sort",
+  "compute_filter",
+  "compute_temporal",
+  "compute_comparison",
+  "compute_arithmetics",
+  "compute_concatenate",
+  "compute_substring",
+  "io_ipc"
+]}
```

Comment:
do we need all of these dependencies?

Reply:
No, I forgot to remove them when I copy and pasted from daft-core.